assets/blocklist.private.json

```diff
@@ -0,0 +1,3 @@
+{
+    "numbers": ["+12025550102"]
```

Comment:
This is included in the PR, but is also in `.gitignore`.  Did you mean to `git rm` this?